### PR TITLE
move functionality

### PR DESCRIPTION
**Pull request by supreme2580**

### CODE REVIEW

File: client/LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2023 Dojo
+Copyright (c) Bibliotheca DAO
```
Comment (okhaimie-dev) [inline client/LICENSE:3]: @supreme2580 why did you change this copyright to Bibliotheca?

Please change to Realm of Ra instead

File: client/readme.md [hunk 2]
```diff
@@ -1 +1,65 @@
-## React-app
+<div align="center">
```
Comment (okhaimie-dev) [inline client/readme.md:1]: this is already on the root directory. if you want to add client specific readme, create something unique and more specific to the client, or take this out.

File: client/src/dojo/generated/setup.ts
Comment (okhaimie-dev) [inline client/src/dojo/generated/setup.ts:1]: we need to keep a consistent format. once this PR is merge we can go over this, and run a format to make consistent.

File: client/readme.md (hunk 2 above)
Comment (supreme2580) [reply client/readme.md]: Removed this from client directory

**Review by okhaimie-dev — APPROVED** [review]
